deploy/olm-catalog/operand-deployment-lifecycle-manager/1.1.0/operand-deployment-lifecycle-manager.v1.1.0.clusterserviceversion.yaml

```diff
@@ -62,8 +62,7 @@ metadata:
                 "name": "ibm-commonui-operator",
                 "spec": {
                   "commonWebUI": {},
-                  "legacyHeader": {},
-                  "navconfiguration": {}
+                  "legacyHeader": {}
```

Comment:
why doing this change? the v1.1.0 is in stable-v1 channel, we should not update it.

And this change does not impact the 3.3 release as we use yaml file to create the operandconfig.

Please rollback this change.

Reply:
I roll it back